addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -5484,7 +5484,7 @@ QUnit.module("Views", (hooks) => {
         await click(target.querySelector(".o_pager_next"));
 
         assert.strictEqual(target.querySelector(".o_pager_value").textContent, "2");
-        assert.verifySteps(["get_views", "web_read", "web_save", "web_read"]);
+        assert.verifySteps(["get_views", "web_read", "web_save"]);
```

Comment:
now that we do not have the last web_read, it would be nice to assert that the record has been correctly loaded (assert foo value)